GAME/CONTROL.C

```diff
@@ -34,6 +34,7 @@
 #include "TOMB4FX.H"
 
 #include <assert.h>
+#include "LOT.H"
```

Comment:
Non c library includes should be grouped together.

Reply:
fixed